service/src/prover_service.rs

```diff
@@ -439,16 +486,31 @@ impl ProverHandler for ProverRequestHandler {
     ) -> Result<ProverResponse> {
         let task_id = match PIPELINE.lock().unwrap().final_prove(
             request.recursive_proof.clone(),
-            request.curve_name.clone(),
+            // request.curve_name.clone(),
+            "BN128".to_string(),
```

Comment:
Why should we hardcode the curve? we also need BLS12381.

Reply:
ok, we can get from env or config